extra-web/nginx/autobuild/conffiles

```diff
@@ -6,5 +6,6 @@
 /etc/nginx/nginx.conf
 /etc/nginx/scgi_params
 /etc/nginx/uwsgi_params
+/etc/nginx/sites-available/default
 /etc/nginx/win-utf
 /etc/logrotate.d/nginx
```

Comment:
One option would be to just let Autobuild3 handle conffiles generation - drop this file.